src/analysis/inner_product_space/basic.lean

```diff
@@ -1419,6 +1419,41 @@ begin
     ... ≤ ∥innerSL x∥ * ∥x∥ : (innerSL x).le_op_norm _ }
 end
 
+/-- The inner product as a continuous sesquilinear map, with the two arguments flipped. -/
+def innerSL_flip : E →L[𝕜] E →L⋆[𝕜] 𝕜 :=
+continuous_linear_map.flipₗᵢ' E E 𝕜 (ring_hom.id 𝕜) (↑star_ring_aut : 𝕜 →+* 𝕜) innerSL
+
+@[simp] lemma innerSL_flip_apply {x y : E} : innerSL_flip x y = ⟪y, x⟫ := rfl
+
+namespace continuous_linear_map
+
+variables {𝕜₂ : Type*} {E' : Type*} {G : Type*}
+variables [is_R_or_C 𝕜₂] [inner_product_space 𝕜 E'] [inner_product_space 𝕜₂ G]
+variables {σ : 𝕜₂ →+* 𝕜} [ring_hom_isometric σ]
+
+/-- Given `f : E →L[𝕜] E'`, construct the continuous sesquilinear form `λ x y, ⟪x, A y⟫`, given
+as a continuous linear map. -/
+def to_sesq_form : (E →L[𝕜] E') →L[𝕜] E' →L⋆[𝕜] E →L[𝕜] 𝕜 :=
+↑((continuous_linear_map.flipₗᵢ' E E' 𝕜
+  (↑(star_ring_aut : 𝕜 ≃+* 𝕜) : 𝕜 →+* 𝕜) (ring_hom.id 𝕜)).to_continuous_linear_equiv) ∘L
+(continuous_linear_map.compSL E E' (E' →L⋆[𝕜] 𝕜) (ring_hom.id 𝕜) (ring_hom.id 𝕜) innerSL_flip)
+
+@[simp] lemma to_sesq_form_apply_coe (f : E →L[𝕜] E') (x : E') :
+  (to_sesq_form f x : E →L[𝕜] 𝕜) = (innerSL x).comp f := rfl
```

Comment:
```suggestion
  to_sesq_form f x = (innerSL x).comp f := rfl
```
I think the form of the lemma suggests that there's a coercion happening, but there's not.